src/main/java/edu/harvard/iq/dataverse/branding/BrandingUtil.java

```diff
@@ -1,22 +1,37 @@
 package edu.harvard.iq.dataverse.branding;
 
+import edu.harvard.iq.dataverse.DataverseServiceBean;
 import edu.harvard.iq.dataverse.util.BundleUtil;
 import java.util.Arrays;
+
+import javax.ejb.EJB;
+import javax.ejb.Singleton;
 import javax.mail.internet.InternetAddress;
 
+@Singleton
 public class BrandingUtil {
 
-    public static String getInstallationBrandName(String rootDataverseName) {
-        return rootDataverseName;
+    @EJB
+    static DataverseServiceBean dataverseService;
+    
+    public static String getInstallationBrandName() {
```

Comment:
Is there a particular reason why this is still all static methods? This is an injectable bean now, so does it matter?

If we still want to have static methods, maybe we can do the injection the same way I did with `DBSettingsSource`? (A helper bean, injecting the service on construction into the POJO)

Reply:
I've gone back to using a helper - I don't think it was working correctly (but a little hard to tell given other problems - to be discussed elsewhere). W.r.t. static methods - a) there is no state, and b) since there's no discovery mechanism as with the ConfigSources, it would mean making this a bean (it isn't one with the helper) and injecting many places. There's a closed PR where I started looking into how much would have to change for that - this looked better.)